pandas_decimal/array.py

```diff
@@ -180,7 +180,9 @@ def take(self, indices, *, allow_fill=False, fill_value=None):
         return self[indices]
 
     def copy(self):
-        return type(self)(np.copy(self._data), dtype=self._dtype)
+        return type(self)(
+            np.copy(self._data / 10**self.dtype.decimal_places), dtype=self._dtype
```

Comment:
This one looks wrong, shouldn't the new extension array have exactly the same data as the original? If it corrects the data but keeps the same number of decimal places in the dtype, you end up correcting twice.

Reply:
ah right, should have been
```
return type(self).from_internal(np.copy(self._data), dtype=self._dtype)
```
(this involves no compute)

Reply:
This is now added.